ARMeilleure/Instructions/InstEmitSimdHelper.cs

```diff
@@ -234,6 +234,20 @@ public static int X86GetRoundControl(FPRoundingMode roundMode)
             throw new ArgumentException($"Invalid rounding mode \"{roundMode}\".");
         }
 
+        public static Operand EmitCountSetBits8(ArmEmitterContext context, Operand me) // "size" is 8 (SIMD&FP Inst.).
+        {
+            Operand me0 = context.Subtract(me, context.BitwiseAnd(context.ShiftRightUI(me, Const(1)), Const(me.Type, 0x55L)));
+
+            Operand c1 = Const(me.Type, 0x33L);
+            Operand me1 = context.Add
+            (
+                context.BitwiseAnd(context.ShiftRightUI(me0, Const(2)), c1),
+                context.BitwiseAnd(me0, c1)
+            );
+
+            return context.BitwiseAnd(context.Add(me1, context.ShiftRightUI(me1, Const(4))), Const(me.Type, 0x0fL));
+        }
```

Comment:
```suggestion
        public static Operand EmitCountSetBits8(ArmEmitterContext context, Operand op) // "size" is 8 (SIMD&FP Inst.).
        {
            Debug.Assert(op.Type == OperandType.I32 || op.Type == OperandType.I64);

            Operand op0 = context.Subtract(op, context.BitwiseAnd(context.ShiftRightUI(op, Const(1)), Const(op.Type, 0x55L)));

            Operand c1 = Const(op.Type, 0x33L);
            Operand op1 = context.Add(context.BitwiseAnd(context.ShiftRightUI(op0, Const(2)), c1), context.BitwiseAnd(op0, c1));

            return context.BitwiseAnd(context.Add(op1, context.ShiftRightUI(op1, Const(4))), Const(op.Type, 0x0fL));
        }
```
Now that it's a shared method, I'd rename the `m` to `op`; I'd avoid that `Add()` on multiple lines; but most importantly I'd add a `Debug.Assert()` like that just because that shared method was designed for those value types.